### PR TITLE
feature(backend) setup backend to receive urls

### PR DESCRIPTION
**Pull request by Bansikah**
feature-55 setup backend script to receive urls (Nancy and Noel)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -10,9 +10,30 @@ def index():
 def input():
 	return render_template("input.html")
 
+#setting up backend to receive urls
```
Comment (stephane-segning) [inline app.py:13]: Change the name from `/scrape` to something more like `/save_url`.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Very good job. Please update this one here:

**Comment by stephane-segning** [comment]
I pushed some changes to your branch. You'll need to pull it first @Bansikah

File: app.py (hunk 1 above)
Comment (AssahBismarkabah) [reply app.py:15]: good evening @Bansikah  i will just use the` /scrape` for the label for the script to scrap the data from the site then

**Comment by Bansikah** [comment]
all good sir, corrected as specified

**Comment by stephane-segning** [comment]
> all good sir, corrected as specified

Hello @Bansikah, no. I still see the path /scrape. Please change it as I suggested please

**Comment by Bansikah** [comment]
surely did the changes locally and pushed but it didn't come, that's strang

**Review by stephane-segning — APPROVED** [review]
Very good work @Bansikah @AssahBismarkabah

**Pull request merged**